### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by RA489**
Adding license file for logo.
Related to  https://github.com/kubernetes/minikube/issues/3030

### CODE REVIEW

**Comment by idealhack** [comment]
Is this related to kubernetes/minikube#3030?

**Comment by neolit123** [comment]
yep, seems like it.

/reopen
/assign @thockin

File: logo/LICENSE [hunk 1]
```diff
@@ -0,0 +1 @@
+# The kubernetes logo files are licensed under a choice of either Apache-2.0 or CC-BY-4.0 (Creative Commons Attribution 4.0 International).
```
Comment (idealhack) [inline logo/LICENSE]: k -> K

**Review by idealhack — CHANGES_REQUESTED** [review]

File: logo/LICENSE (hunk 1 above)
Comment (idealhack) [reply logo/LICENSE]: @RA489 Thank you! Could you please squash the commits to one?

**Comment by RA489** [comment]
@idealhack I have squashed the commits to one.

**Review by idealhack — APPROVED** [review]
/lgtm

**Pull request merged**